### ISSUE TITLE
AMP compatibility

### ISSUE THREAD

**Issue by rohittm** [Codeinwp/wp-product-review#254]
@selul I had discussions with @contactashish13 and @HardeepAsrani about how we fixed the issues with AMP compatibility and passed the validation. Since then, we had new designs and the AMP plugin also has new templates which is not passing the AMP validations. So, the compatibility we added was just for old AMP template, and our old design. 

The AMP plugin will keep on updating it's features and we will keep on updating ours. AMP's limitation will stay to just <=50000 bytes. So, what I suggest if we have just a basic template (i.e. with minimal CSS, no custom icons, and circle ratings) just for AMP. @contactashish13 also suggested if we can have the template like Top/Latest widget's style1 with Pros, and Cons. 

So, Instead of spending a huge amount of time on fixing and Validating AMP compatibilities, we can have this basic template, when user is using AMP. What do you think?

P.S. The new features, and fixes (of WPPR), will always work with the non-AMP version.

**Comment by selul** [comment]
yep, that's what i tried to solve with amp-style css. The ideal case will be to keep consistency between how the reviews will be looking on amp and non-amp versions.
